.github/workflows/publish.yaml

```diff
@@ -40,10 +37,9 @@ jobs:
 
       - name: Import and push to Docker Hub
         run: |
-          version="$(cat rockcraft.yaml | yq e '.version')"
           sudo skopeo \
             --insecure-policy \
             copy \
-            oci-archive:"mongodb_${version}_amd64.rock" \
-            docker-daemon:"dataplatformoci/mongodb:${version}"
-          docker push "dataplatformoci/mongodb:${version}"
+            oci-archive:charmed-mongodb_${{ github.ref_name }}_amd64.rock \
+            docker-daemon:dataplatformoci/charmed-mongodb:${{ github.ref_name }}
+          docker push dataplatformoci/charmed-mongodb:${{ github.ref_name }}
```

Comment:
As a follow up review, move move this to GitHub repo (so we will have all puzzle pieces in places). Tnx!